test/core/util/evaluate_args_test_util.h

```diff
@@ -34,7 +34,8 @@ class EvaluateArgsTestUtil {
 
   void AddPairToMetadata(const char* key, const char* value) {
     metadata_.Append(
-        key, Slice(grpc_slice_intern(grpc_slice_from_static_string(value))));
+        key, Slice(grpc_slice_intern(grpc_slice_from_static_string(value))),
+        [](absl::string_view, const Slice&) {});
```

Comment:
Maybe have this lambda do an `ASSERT_EQ("", error)`, so that the test will fail if there's a problem here?

Reply:
done

Reply:
(although made it an abort - we should not see this function called).